docs/source/man.rst

```diff
@@ -302,7 +302,7 @@ Scanning
 Use the ``qpc scan start`` command to run connect and inspection scans on the sources. This command scans all of the host names or IP addresses that are defined in the supplied source, and then writes the report information to a comma separated values (CSV) file. Note: Any ssh-agent connection setup for a target host '
               'will be used as a fallback if it exists.
 
-**qpc scan start --sources=** *source_list* **[--max-concurrency=** *concurrency* **]**
+**qpc scan start --sources=** *source_list* **[--max-concurrency=** *concurrency* **]** **--disable-optional-products=** *products_list*
 
```

Comment:
@chambridge When I added disable optional products to line 305, do I need to add the options instead of just product list ie. `*[jboss_eap, jboss_fuse, jboss_brms]*` or do I leave it as `*products_list*` ? Thank you!

Reply:
@chambridge did you get a chance to look at this? Thank you!